composer.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "apitte/console",
+  "minimum-stability": "dev",
```

Comment:
Please reformat composer.json according to the others. (https://github.com/apitte/openapi/blob/master/composer.json)

Reply:
Remove composer.lock.